docs/content/any/project/changelogs/2021-05-26.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Release 2021-05-26
+menuTitle: 2021-05-26
+any: true
+description: >-
+  Changelog for Release 2021-05-26 (sqlalchemy-oso-preview 0.0.4) containing new features, bug fixes, and more.
+---
+
+## `sqlalchemy-oso-preview` 0.0.4
+
+### Breaking changes
+
+{{% callout "Warning" "orange" %}}
+  This release contains breaking changes. Be sure to follow migration steps
+  before upgrading.
+{{% /callout %}}
+
+#### Performance
+
+Updates to generated sql and roles tables to better utilize indexes.
```

Comment:
This is kind of confusing because most people probably don't even know we generate SQL. I'd maybe just title this section "Resource model primary key requirements" and remove this sentence, unless we have concrete performance benchmarks to include that show how performance is better.